image_services/egis/metadata/templates/egis_template.yaml.j2

```diff
@@ -0,0 +1,88 @@
+concept_id: None
+collection: None
+data_provider: ASF
+
+doi: https://doi.org/10.5067/8W33RRS6S2RV
+cmr: None
+
+egis:
+  id: None
+  phase: None
+  priority: None
+
+service:
+  type: image
+  ogc: None
+
+deployment:
+  sit:
+    status: False
+    notes: None
+    data: None
+  uat:
+    status: False
+    notes: None
+    data: None
+  prd:
+    status: True
+    notes: None
+    data: None
+
+update:
+  live: False
+  period: None
+
+time:
+  start: {{ start_date }}
+  end: {{ end_date }}
+  level: Seasonal
+  range: {{ date_range }}
+  standard: UTC
+
+spatial:
+  extent: Global
+  projection: WGS84
+  resolution: 3 arcsecond pixel spacing
+
+eula:
+  type: False
+  notes: None
+
+portal:
+  name: GSSICB Median {{ interval }}-day {{ polarization }} Coherence {{ months_abbreviated }}
+  summary: Global {{ season }} ({{ months_full }}) median {{ interval }}-day interferometric coherence ({{ data_type }}) in {{ polarization }} polarization from the Global Seasonal Sentinel-1 Interferometric Coherence and Backscatter Data Set.
+  description: |
+    <div></div><div><br/></div><div><b>Time Interval: </b>Seasonal</div><div><b>Time Extent: </b>{{ start_date }} to {{ end_date }}</div><div><b>Time Standard: </b>Coordinated Universal Time (UTC)</div><div><b>Grid Size: </b>3 arcsecond pixel spacing</div><div><b>Projection: </b>WGS84</div><div><b>Extent: </b>Global<br /></div><div><b>Source:</b>Alaska Satellite Facility (ASF)</div><div><br/></div><div><div><b>Data Sources:</b></div><div><ul> <li>https://doi.org/10.5067/8W33RRS6S2RV</li></ul></div></div><div><div><b>Data Parameters:</b></div><div>For questions or issues please email: <a href="mailto:uso@asf.alaska.edu" target="_blank">uso@asf.alaska.edu</a></div></div>
+  terms_of_use: |
+    <div><font color="#000000"><b><a href="https://www.earthdata.nasa.gov/data-and-information-policy">NASA Earth Science Data and Information Policy</a></b></font></div><div><br/></div><div><font color="#000000"><b>Data Use: </b>When the data products are used in a publication, we request the following acknowledgment be included: <i>&quot;These data were obtained from the Alaska Satellite Facility (ASF).&quot;</i></font></div><div><br/></div><div><font color="#000000"><b>Acknowledgements: </b>This service is provided by <a href="https://gis.earthdata.nasa.gov">Earthdata GIS (EGIS)</a> supported by <a href="https://www.earthdata.nasa.gov/">Earth Observing System Data and Information System (EOSDIS)</a>. We would like to acknowledge the science teams which have provided input and validation throughout service creation process.</font></div>
+  tags:
+    - Synthetic Aperture Radar
+    - SAR
+    - Coherence
+    - Sentinel-1
+    - Alaska Satellite Facility
+    - ASF
+  credits: Global Seasonal Sentinel-1 Interferometric Coherence and Backscatter Data Set was accessed on 11 November 2022 from https://registry.opendata.aws/ebd-sentinel-1-global-coherence-backscatter.
```

Comment:
We need to make the accessed date interactive, now that we're generating services with our new script. This also needs to be changed in for our templates for populating mosaic dataset metadata; for the COH06 services, I made the change manually in the portal.

It was formerly hard-coded, because all of the MDCS-generated mosaic datasets were generated on the same day, but we should update this so that it's a dynamic field populated with the date on which the script was run (assuming we're generating the metadata at the same time as we're generating the mosaic dataset).

Reply:
(I didn't notice it until after the services had been published, so the change is not reflected in the metadata files for the COH06 services)